tests/unit/test_firewall_lib.py

```diff
@@ -809,6 +816,18 @@ def test_nm_integration_interfaces(
                 firewall_lib.main()
```

Comment:
```suggestion
                fw_settings.queryService.return_value = state == "disabled"
                firewall_lib.main()
```
sorry - looks like I previously posted the wrong diff :-(